NorthstarLauncher/main.cpp

```diff
@@ -156,7 +156,7 @@ void AwaitOriginStartup()
 			recv(sock, buf, 4096, 0);
 			std::cout << buf << std::endl;
 
-			// honestly really shit, this isn't needed for origin due to being able to check OriginClientService
```

Comment:
this is inaccurate, we handle ea app and origin differently, this check IS necessary for ea app, but isn't for origin, hence the original comment